Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -186,38 +224,19 @@ void main()
     float pixelOffsetScale = getNearFarScalar(pixelOffsetScaleByDistance, lengthSq);
     pixelOffset *= pixelOffsetScale;
 #endif
-
-    vec4 positionWC = czm_eyeToWindowCoordinates(positionEC);
-    
-    vec2 halfSize = imageSize * scale * czm_resolutionScale;
-    halfSize *= ((direction * 2.0) - 1.0);
-    
-    positionWC.xy += (origin * abs(halfSize));
     
-#if defined(ROTATION) || defined(ALIGNED_AXIS)
-    if (!all(equal(alignedAxis, vec3(0.0))) || rotation != 0.0)
+#ifdef TEST_GLOBE_DEPTH
+    vec4 offsetPosition = positionEC + vec4(0.0, 0.0, -positionEC.z * 0.05, 0.0);
+    vec4 wc = computePositionWindowCoordinates(offsetPosition, vec2(0.0, 0.0), scale, direction, origin, vec2(0.0), pixelOffset, alignedAxis, rotation);
+    float d = texture2D(czm_globeDepthTexture, wc.xy / czm_viewport.zw).r;
+    if (wc.z > d)
     {
-        float angle = rotation;
-        if (!all(equal(alignedAxis, vec3(0.0))))
-        {
-            vec3 pos = positionEC.xyz + czm_encodedCameraPositionMCHigh + czm_encodedCameraPositionMCLow;
-            vec3 normal = normalize(cross(alignedAxis, pos));
-            vec4 tangent = vec4(normalize(cross(pos, normal)), 0.0);
-            tangent = czm_modelViewProjection * tangent;
-            angle += sign(-tangent.x) * acos(tangent.y / length(tangent.xy));
-        }
-        
-        float cosTheta = cos(angle);
-        float sinTheta = sin(angle);
-        mat2 rotationMatrix = mat2(cosTheta, sinTheta, -sinTheta, cosTheta);
-        halfSize = rotationMatrix * halfSize;
+        gl_Position = czm_projection * vec4(vec3(0.0), 1.0);
```

Comment:
Do we even need the projection matrix multiplication to cull here?  Can't we just assign a value in clip coordinates that will be behind the near plane?